roles/openshift_logging/tasks/install_fluentd.yaml

```diff
@@ -5,6 +5,10 @@
 - set_fact: fluentd_ops_port={{ (openshift_logging_use_ops | bool) | ternary(openshift_logging_es_ops_port, openshift_logging_es_port) }}
   check_mode: no
 
+- set_fact: openshift_logging_fluentd_use_journal={{ "journald" in openshift.docker.options }}
+  check_mode: no
+  when: "openshift_logging_fluentd_use_journal == ''"
```

Comment:
Is it worth having it as a configurable option then?
This is to satisfy  https://bugzilla.redhat.com/show_bug.cgi?id=1401417
Effectively we're doing the same logic as would be done within the Fluentd pod, correct?